### PR TITLE
Add explanation about time zones to Docker instructions

### PR DESCRIPTION
**Pull request by citelao**
## Proposed change

On the `docker run` installation instructions, add an example time zone value and a link to the list of time zones on Wikipedia.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - ✅ I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @citelao, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20277).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_includes/installation/container.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ If you are using Docker then you need to be on at least version 19.03.9, ideally
 
 ### Platform Installation
 
-Installation with Docker is straightforward. Adjust the following command so that `/PATH_TO_YOUR_CONFIG` points at the folder where you want to store your configuration and run it.
+Installation with Docker is straightforward. Adjust the following command so that:
+
+* `/PATH_TO_YOUR_CONFIG` points at the folder where you want to store your configuration and run it.
+* `MY_TIME_ZONE` is [tz database name](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), like `TZ=America/Los_Angeles`.
```
Comment (citelao) [inline source/_includes/installation/container.md]:
```suggestion
* `MY_TIME_ZONE` is a [tz database name](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), like `TZ=America/Los_Angeles`.
```

**Comment by homeassistant** [comment]
Hi @citelao, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20277).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @citelao 👍

**Pull request merged**